README.md

```diff
@@ -782,6 +782,17 @@ If using a `json` file, the format should be:
 }
 ```
 
+If you want to retry the tuples with errors, you can direct the output to a file:
+
+
+`fga tuple write --file tuples.json' --hide-imported-tuples > results.json`
+
+And then process the file with `jq` to convert it to format that you can send the CLI again:
+
+`jq -c '[.failed[] | {user: .tuple_key.user, relation: .tuple_key.relation, object: .tuple_key.object}]' result.json > failed_tuples.json`
```

Comment:
Given that a write failure isn't always retryable (i.e existing tuple, invalid tuple etc.) do we not want to try and guide people to review these prior to this and maybe provide some examples of what can be retried?

Reply:
Good point, I rephrased it, let me know what you think